tests/lib/data/alignment.test.js

```diff
@@ -58,38 +58,37 @@ describe('alignment.test.js', () => {
   })
 
   it('3 Alignment - readyForTokenize return true if origin docSource and target docSource are defined with all obligatory fields (text, direction, lang)', () => {
-    const sourceTextOrigin = new SourceText('origin', {
-      text: 'origin some text', direction: 'ltr', lang: 'lat'
+    const originDocSource = new SourceText('origin', {
+      text: 'origin some text', direction: 'ltr', lang: 'eng', sourceType: 'text', tokenization: { tokenizer: "simpleLocalTokenizer" }
     })
-
-    const sourceTextTarget = new SourceText('target', {
-      text: 'target some text', direction: 'ltr', lang: 'eng'
+    const targetDocSource = new SourceText('origin', {
+      text: 'target some text', direction: 'ltr', lang: 'eng', sourceType: 'text', tokenization: { tokenizer: "simpleLocalTokenizer" }
     })
 
     let alignment = new Alignment()
 
     expect(alignment.readyForTokenize).toBeFalsy()
 
-    alignment.updateOriginDocSource(sourceTextOrigin)
+    alignment.updateOriginDocSource(originDocSource)
 
     expect(alignment.readyForTokenize).toBeFalsy()
 
-    alignment.updateTargetDocSource(sourceTextTarget)
+    alignment.updateTargetDocSource(targetDocSource)
 
     expect(alignment.readyForTokenize).toBeTruthy()
   })
 
   it('4 Alignment - equalSegmentsAmount checks if all aligned texts have the same amount of segments', async () => {
     const originDocSource1 = new SourceText('origin', {
-      text: 'some text', direction: 'ltr', lang: 'eng'
+      text: 'origin some text', direction: 'ltr', lang: 'eng', sourceType: 'text', tokenization: { tokenizer: "simpleLocalTokenizer" }
     })
 
-    const targetDocSource1 = new SourceText('target', {
-      text: 'some text', direction: 'ltr', lang: 'eng'
+    const targetDocSource1 = new SourceText('origin', {

```

Comment:
should this be 'target' ? Does it matter?

Reply:
oh, missed that when was updating tests - fixed